src/Schema/Events/Click.ts

```diff
@@ -202,7 +206,9 @@ export interface ClickedArtworkGroup extends ClickedEntityGroup {
  *  }
  * ```
  */
-export interface ClickedAuctionGroup extends ClickedEntityGroup {
+export interface ClickedAuctionGroup
+  extends ClickedEntityGroup,
+    CollectorSignalsValues {
```

Comment:
Ditto here and elsewhere. We can surely achieve the same ends without crossing wires.